students/7/modern-js.js

```diff
@@ -6,15 +6,15 @@
 //       - Replace var with let or const.
 //
 
-function displayResults(personList, count) {
```

Comment:
Does  `count` have to be in this function signature?

Reply:
I was able to fix it!